packages/react/slider/src/Slider.tsx

```diff
@@ -88,6 +90,7 @@ const Slider = React.forwardRef<SliderElement, SliderProps>(
       onValueChange = () => {},
       onValueCommit = () => {},
       inverted = false,
+      form,
```

Comment:
A slider can render multiple thumbs with separate inputs. Form attribute should be shared, so lifting this one to the root.